.github/workflows/builds.yml

```diff
@@ -2130,7 +2130,8 @@ jobs:
       - run: CFLAGS="${DEFAULT_CFLAGS}" ./configure --enable-unittests
       - run: make -j2
       - run: make check
-      - run: make -j2 distcheck
+        # -j2 caused random failures during cargo vendor
```

Comment:
Wouldn't it be better to have a comment closer to the `Parallel builds` mention from the commit? It feels that it would be more informative.

Reply:
`-j` is parallel builds, so this is just a compressed version